kibana/fleet.go

```diff
@@ -321,6 +321,11 @@ type AgentCommon struct {
 		Host struct {
 			Hostname string `json:"hostname"`
 		} `json:"host"`
+		Elastic struct {
+			Agent struct {
+				FIPS bool `json:"fips"`
```

Comment:
👍 added to ES via https://github.com/elastic/elasticsearch/pull/123406

Have we confirmed that it's a part of the kibana-fleet API?

Reply:
> Have we confirmed that it's a part of the kibana-fleet API?

I spun up an ECH `9.1.0-SNAPSHOT` deployment on May 1 and I confirmed by making a `GET kbn:/api/fleet/agents` call from the Kibana Dev Tools Console that the response contains this field.